playbook/dist/reset.css

```diff
@@ -1 +1,61 @@
-*{box-sizing:border-box;margin:0;padding:0}*:before,*:after{box-sizing:border-box}html{-webkit-tap-highlight-color:rgba(0,0,0,0);height:100vh;overflow-x:hidden}body{font-family:"Proxima Nova","Helvetica Neue",Helvetica,Arial,sans_serif;font-size:16px;line-height:1.5;background-color:#F3F7FB;height:100%;letter-spacing:0;font-weight:400;font-style:normal;text-rendering:optimizeLegibility;-moz-font-feature-settings:"liga" on;color:#242B42;margin:0 !important;padding:0 !important;box-sizing:border-box;min-height:100vh;padding:50px}a{text-decoration:none;color:#0056CF}
+/* CLEAN UP AND REMOVE */
```

Comment:
I guess this gets cleaned up with a release?

Reply:
Yeah this is a strange one.  Meant to bring it up at dailies.

The new repo setup re-compiles the dist when you make changes.  It also re-compiles the `reset.css`, but probably doesn't need to.  Sometimes webpack refused to compile because of some problem with `reset.css`, then I'd have to restart the server.  Didn't have the time to search through webpack config to track down what's going on.

TLDR: we may need to take a look at what's getting compiled when.